templates/server.conf.erb

```diff
@@ -55,6 +65,8 @@ ssl_certfile: <%= scope['pulp::db_ssl_certfile'] %>
 verify_ssl: <%= scope['pulp::db_verify_ssl'] %>
 ca_path: <%= scope['pulp::db_ca_path'] %>
 <% end -%>
+# unsafe_autoretry: false
+# write_concern: majority
```

Comment:
so i had discussed this a bit with partha.  there are existing settings that do not have parameters and are commented out (https://github.com/Katello/puppet-pulp/blob/master/templates/server.conf.erb#L166-L206), and partha wanted to make this file look as much like the stock server.conf as possible hence these entries.  I assumed we could add parameters for all of them either in a followup PR or as needed.   Ideally every config value would be configurable but that already isn't the case, so is it that we prefer:

a) any new config entires added should have parameters and should be 100% up to date with the corresponding pulp stock configs
b) config entires shouldn't be added until there are parameters, meaning some may be missing
c) $$$$

Trying to understand myself for future contributions :)